Exadel.Compreface.AcceptenceTests/Services/SubjectExampleServiceTest.cs

```diff
@@ -0,0 +1,254 @@
+
+            var actualSubjectExample = resultList.Faces
+                .First(x => x.ImageId == expectedAddExampleSubjectResponse.ImageId & x.Subject == expectedAddExampleSubjectResponse.Subject);
+
+            //Assert
+            Assert.Equal(expectedAddExampleSubjectResponse.Subject, actualSubjectExample.Subject);
+            Assert.Equal(expectedAddExampleSubjectResponse.ImageId, actualSubjectExample.ImageId);
+
+            await comprefaceApiClient.SubjectExampleService.ClearSubjectAsync(new DeleteAllExamplesRequest() { Subject = addNewSubjectResponse.Subject });
+        }
+
+        [Fact]
+        public async Task AddBase64SubjectExampleAsync_TakesRequestModel_ReturnsProperResponseModel()
+        {
+            //Arrange
+            var addNewSubjectResponse = await comprefaceApiClient.SubjectService.AddSubject(new AddSubjectRequest() { Subject = TEST_SUBJECT_NAME });
+            var addBase64ExampleSubjectRequest = new AddBase64SubjectExampleRequest()
+            {
+                DetProbThreShold = 0.81m,
+                Subject = addNewSubjectResponse.Subject,
+                File = IMAGE_BASE64_STRING
+            };
+
+            var expectedAddBase64SubjectExampleResponse = await comprefaceApiClient.SubjectExampleService.AddBase64SubjectExampleAsync(addBase64ExampleSubjectRequest);
+
+            //Act
+            var resultList = await comprefaceApiClient.SubjectExampleService.GetAllSubjectExamplesAsync(new ListAllSubjectExamplesRequest() { Subject = addNewSubjectResponse.Subject });
+            var actualSubjectExample = resultList.Faces
+                    .First(x => x.ImageId == expectedAddBase64SubjectExampleResponse.ImageId & x.Subject == expectedAddBase64SubjectExampleResponse.Subject);
+
+            //Assert
+            Assert.Equal(expectedAddBase64SubjectExampleResponse.Subject, actualSubjectExample.Subject);
+            Assert.Equal(expectedAddBase64SubjectExampleResponse.ImageId, actualSubjectExample.ImageId);
+
+            await comprefaceApiClient.SubjectExampleService.ClearSubjectAsync(new DeleteAllExamplesRequest() { Subject = addNewSubjectResponse.Subject });
+        }
+
+        [Fact]
+        public async Task GetAllSubjectExamplesAsync_TakesRequestModel_ReturnsProperResponseModel()
+        {
+            //Arrange
+            var addNewSubjectResponse = await comprefaceApiClient.SubjectService.AddSubject(new AddSubjectRequest() { Subject = TEST_SUBJECT_NAME });
+            var allSubjectExamples = new ListAllSubjectExamplesRequest()
+            {
+                Page = 0,
+                Size = 0,
+                Subject = addNewSubjectResponse.Subject
+            };
+
+            var actualListAllExampleSubjectResponse = await comprefaceApiClient.SubjectExampleService.GetAllSubjectExamplesAsync(allSubjectExamples);
+            var actualCount = actualListAllExampleSubjectResponse.Faces.Count;
```

Comment:
The variable is never used 🙈

And you're confusing ... 🙃
You expect the method (that you are testing) will return that count of elements
So it's "expected"